manifests/t/tailscale/tailscale/1.4.4/tailscale.tailscale.locale.en-US.yaml

```diff
@@ -0,0 +1,27 @@
+# Created using YamlCreate.ps1 v1.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: tailscale.tailscale
+PackageVersion: 1.4.4
+PackageLocale: en-US
+Publisher: Tailscale Inc.
+#PublisherUrl: 
+PublisherSupportUrl: https://tailscale.com/kb/
+PrivacyUrl: https://tailscale.com/privacy-policy
+#Author: 
+PackageName: Tailscale
+PackageUrl: https://tailscale.com/
+License: BSD 3-Clause
+LicenseUrl: https://github.com/tailscale/tailscale/blob/main/LICENSE
+Copyright: © 2021 Tailscale Inc.
+CopyrightUrl: Copyright (c) 2020 Tailscale & AUTHORS.
```

Comment:
```suggestion
Copyright: Copyright (c) 2020 Tailscale & AUTHORS.
```

Reply:
I had to correct it but left it in between due to some urgent work. Anyways, thanks for the suggestion.